Random panic when launching the app

### Checklist

- [x] I have searched the [issue tracker](https://github.com/fyne-io/fyne/issues) for open issues that relate to the same problem, before opening a new one.
- [x] This issue only relates to a single bug. I will open new issues for any other problems.

### Describe the bug

This happens on both Windows/Linux.

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xc0000005 code=0x0 addr=0x0 pc=0x7ff6ef99793c]

goroutine 1 [running, locked to thread]:
github.com/go-gl/glfw/v3.3/glfw.(*Window).SetInputMode.func1(...)
        /home/joseph/go/pkg/mod/github.com/go-gl/glfw/v3.3/glfw@v0.0.0-20240506104042-037f3cc74f2a/input.go:394
github.com/go-gl/glfw/v3.3/glfw.(*Window).SetInputMode(0x7ff6f0b56360?, 0x33001, 0x34001)
        /home/joseph/go/pkg/mod/github.com/go-gl/glfw/v3.3/glfw@v0.0.0-20240506104042-037f3cc74f2a/input.go:394 +0x1c
fyne.io/fyne/v2/internal/driver/glfw.(*window).processMouseMoved(0xc00059d1e0, 0x7ff6f16cf9c0?, 0xc001b7fa08?)
        /home/joseph/go/pkg/mod/fyne.io/fyne/v2@v2.6.3/internal/driver/glfw/window.go:346 +0x217
fyne.io/fyne/v2/internal/driver/glfw.(*window).mouseMoved(...)
        /home/joseph/go/pkg/mod/fyne.io/fyne/v2@v2.6.3/internal/driver/glfw/window_desktop.go:403
github.com/go-gl/glfw/v3.3/glfw.goCursorPosCB(0xc001b7fa70?, 0x4088680000000000, 0x4079c00000000000)
        /home/joseph/go/pkg/mod/github.com/go-gl/glfw/v3.3/glfw@v0.0.0-20240506104042-037f3cc74f2a/input.go:339 +0x3e
github.com/go-gl/glfw/v3.3/glfw._Cfunc_glfwPollEvents()
        _cgo_gotypes.go:1544 +0x45
github.com/go-gl/glfw/v3.3/glfw.PollEvents()
        /home/joseph/go/pkg/mod/github.com/go-gl/glfw/v3.3/glfw@v0.0.0-20240506104042-037f3cc74f2a/window.go:931 +0x13
fyne.io/fyne/v2/internal/driver/glfw.(*gLDriver).pollEvents(...)
        /home/joseph/go/pkg/mod/fyne.io/fyne/v2@v2.6.3/internal/driver/glfw/loop_desktop.go:22
fyne.io/fyne/v2/internal/driver/glfw.(*gLDriver).runGL(0xc001b7fe00?)
        /home/joseph/go/pkg/mod/fyne.io/fyne/v2@v2.6.3/internal/driver/glfw/loop.go:152 +0x1aa
fyne.io/fyne/v2/internal/driver/glfw.(*gLDriver).Run(0xc0003a6bb0)
        /home/joseph/go/pkg/mod/fyne.io/fyne/v2@v2.6.3/internal/driver/glfw/driver.go:162 +0x72
fyne.io/fyne/v2/app.(*fyneApp).Run(0xc0003a6d10)
        /home/joseph/go/pkg/mod/fyne.io/fyne/v2@v2.6.3/app/app.go:77 +0x102
...client_ui.(*fyneApp).Run(0xc0003788c0?)
        /app/main.go:53 +0x1b
main.main()
        /app/main.go:49 +0x4c2
```

### How to reproduce

Sorry, but I'm having a hard time finding a way to reproduce it minimally.

### Screenshots

_No response_

### Example code

Sorry, but I'm having a hard time finding a way to reproduce it minimally.

### Fyne version

2.6.3

### Go compiler version

1.25.0

### Operating system and version

Windows 11, Ubuntu 24.04.1

### Additional Information

_No response_

```log
2026/01/03 13:11:40 utils.go:41: [Utils] Running from directory: C:\Program Files (x86)\Steam\steamapps\common\Medieval II Total War\mods\ago_beta
2026/01/03 13:11:40 updater.go:112: [Updater] Checking for updates...
2026/01/03 13:11:40 updater.go:40: [Updater] Retrieving mod version
2026/01/03 13:11:40 updater.go:64: [Updater] Mod version 3.3.3 found
2026/01/03 13:11:40 updater.go:72: [Updater] Retrieving latest mod version
2026/01/03 13:11:40 updater.go:104: [Updater] Found latest version 3.3.4
2026/01/03 13:11:40 updater.go:118: [Updater] !!! New mod version found !!!
2026/01/03 13:11:40 updater.go:119: [Updater] Current Version:  3.3.3
2026/01/03 13:11:40 updater.go:120: [Updater] Latest Version:  3.3.4
2026/01/03 13:11:40 config.go:33: [Config] Attempting to open config file at full path: C:\Program Files (x86)\Steam\steamapps\common\Medieval II Total War\mods\ago_beta\AGO.cfg
2026/01/03 13:11:40 config.go:61: [Config] Parsing ini config file at C:\Program Files (x86)\Steam\steamapps\common\Medieval II Total War\mods\ago_beta\AGO.cfg to struct
2026/01/03 13:11:40 config.go:33: [Config] Attempting to open config file at full path: C:\Program Files (x86)\Steam\steamapps\common\Medieval II Total War\mods\ago_beta\TATW.cfg
2026/01/03 13:11:40 config.go:61: [Config] Parsing ini config file at C:\Program Files (x86)\Steam\steamapps\common\Medieval II Total War\mods\ago_beta\TATW.cfg to struct
2026/01/03 13:11:40 config.go:33: [Config] Attempting to open config file at full path: C:\Program Files (x86)\Steam\steamapps\common\Medieval II Total War\mods\ago_beta\eopData\config\gameCfg.json
2026/01/03 13:11:40 config.go:47: [Config] Parsing json config file at C:\Program Files (x86)\Steam\steamapps\common\Medieval II Total War\mods\ago_beta\eopData\config\gameCfg.json to struct
2026/01/03 13:11:40 config.go:33: [Config] Attempting to open config file at full path: C:\Program Files (x86)\Steam\steamapps\common\Medieval II Total War\mods\ago_beta\eopData\config\battlesCfg.json
2026/01/03 13:11:40 config.go:47: [Config] Parsing json config file at C:\Program Files (x86)\Steam\steamapps\common\Medieval II Total War\mods\ago_beta\eopData\config\battlesCfg.json to struct
2026/01/03 13:11:40 news.go:31: [News] Starting GetNewsItems
2026/01/03 13:11:40 news.go:59: [News] Found 2 news items
2026/01/03 13:11:40 news.go:64: [News] Processing news item 0: We love our testers!
2026/01/03 13:11:40 news.go:71: [News] Fetching markdown text from URL: https://raw.githubusercontent.com/EddieEldridge/ago-launcher/refs/heads/main/src/resources/newsItems/newsItem1.md
2026/01/03 13:11:40 news.go:77: [News] Successfully fetched markdown text for: We love our testers!
2026/01/03 13:11:40 news.go:64: [News] Processing news item 1: 3.3.0-BETA is now available
2026/01/03 13:11:40 news.go:71: [News] Fetching markdown text from URL: https://raw.githubusercontent.com/EddieEldridge/ago-launcher/refs/heads/main/src/resources/newsItems/newsItem2.md
2026/01/03 13:11:40 news.go:77: [News] Successfully fetched markdown text for: 3.3.0-BETA is now available
2026/01/03 13:11:40 news.go:81: [News] Finished GetNewsItems
2026/01/03 13:11:40 quotes.go:61: [Quoter] Getting random quote
2026/01/03 13:11:40 quotes.go:28: [Quoter] Loading quotes.json
2026/01/03 13:11:40 quotes.go:55: [Quoter] Found 153 quotes
2026/01/03 13:11:40 quotes.go:73: [Quoter] Found random quote: From the havens of Harad, ships of war put out to sea.
2026/01/03 13:11:41 laa.go:19: [LAA] Checking for LAA: C:\Program Files (x86)\Steam\steamapps\common\Medieval II Total War\medieval2.exe
2026/01/03 13:11:41 laa.go:29: [LAA] Checked: C:\Program Files (x86)\Steam\steamapps\common\Medieval II Total War\medieval2.exe
2026/01/03 13:11:41 gui.go:63: [Utils] LAA applied: true
2026/01/03 13:11:41 main.go:25: error: runtime error: invalid memory address or nil pointer dereference
goroutine 1 [running, locked to thread]:
runtime/debug.Stack()
	runtime/debug/stack.go:26 +0x5e
main.main.func1()
	ago-launcher/main.go:24 +0x34
panic({0x7ff6d3cee120?, 0x7ff6d3bed0f0?})
	runtime/panic.go:792 +0x132
github.com/go-gl/glfw/v3.3/glfw.(*Window).SetInputMode.func1(...)
	github.com/go-gl/glfw/v3.3/glfw@v0.0.0-20250301202403-da16c1255728/input.go:394
github.com/go-gl/glfw/v3.3/glfw.(*Window).SetInputMode(0x7ff6d3ef3c20?, 0x33001, 0x34001)
	github.com/go-gl/glfw/v3.3/glfw@v0.0.0-20250301202403-da16c1255728/input.go:394 +0x1c
fyne.io/fyne/v2/internal/driver/glfw.(*window).processMouseMoved(0xc00013c000, 0x7ff6d4497500?, 0xc002eb37f8?)
	fyne.io/fyne/v2@v2.7.1/internal/driver/glfw/window.go:353 +0x217
fyne.io/fyne/v2/internal/driver/glfw.(*window).mouseMoved(...)
	fyne.io/fyne/v2@v2.7.1/internal/driver/glfw/window_desktop.go:402
github.com/go-gl/glfw/v3.3/glfw.goCursorPosCB(0xc0000021c0?, 0x4080580000000000, 0x4078600000000000)
	github.com/go-gl/glfw/v3.3/glfw@v0.0.0-20250301202403-da16c1255728/input.go:339 +0x3e
github.com/go-gl/glfw/v3.3/glfw._Cfunc_glfwPollEvents()
	_cgo_gotypes.go:1544 +0x45
github.com/go-gl/glfw/v3.3/glfw.PollEvents()
	github.com/go-gl/glfw/v3.3/glfw@v0.0.0-20250301202403-da16c1255728/window.go:931 +0x13
fyne.io/fyne/v2/internal/driver/glfw.(*gLDriver).pollEvents(...)
	fyne.io/fyne/v2@v2.7.1/internal/driver/glfw/loop_desktop.go:22
fyne.io/fyne/v2/internal/driver/glfw.(*gLDriver).runGL(0x7ff6d2c89f77?)
	fyne.io/fyne/v2@v2.7.1/internal/driver/glfw/loop.go:154 +0x1aa
fyne.io/fyne/v2/internal/driver/glfw.(*gLDriver).Run(0xc000344840)
	fyne.io/fyne/v2@v2.7.1/internal/driver/glfw/driver.go:162 +0x72
fyne.io/fyne/v2/app.(*fyneApp).Run(0xc0003448f0)
	fyne.io/fyne/v2@v2.7.1/app/app.go:77 +0x102
fyne.io/fyne/v2/internal/driver/glfw.(*window).ShowAndRun(0xc0000cf860)
	fyne.io/fyne/v2@v2.7.1/internal/driver/glfw/window.go:217 +0x64
ago-launcher/gui.RenderToolbar({0x7ff6d3f06f40, 0xc0000cf860}, {0x7ff6d3f06f40, 0xc00013c000}, 0xc000138080, 0xc00034bd40, 0xc000515f20, 0xc0002934d0)
	ago-launcher/gui/gui.go:69 +0x98e
ago-launcher/gui.InitGUI({0x7ff6d3f04638, 0xc0003448f0}, {0x7ff6d3f06f40, 0xc00013c000}, 0xc000138080, 0xc00034bd40, 0xc000515f20, 0xc0002934d0)
	ago-launcher/gui/gui.go:33 +0x13f
main.main()
	ago-launcher/main.go:58 +0x2b1
```
Getting the exact same error. If I publish a new version of the app, it happens to maybe 1/5 users. It can work for them if they just keep trying to launch or restarting PC etc.

Here is my main.go https://github.com/Divide-and-Conquer-AGO/ago-launcher/blob/main/src/main.go

**Fyne version**
2.7.0

**Go compiler version**
1.24.3 windows/amd64

**Windows version**
Windows 10 latest

If you need any other info, please let me know.

--- a/internal/driver/glfw/window.go
+++ b/internal/driver/glfw/window.go
@@ -347,11 +347,13 @@ func (w *window) processMouseMoved(xpos float64, ypos float64) {
 		rawCursor, isCustomCursor := fyneToNativeCursor(cursor)
 		w.cursor = cursor
 
-		if rawCursor == nil {
-			w.view().SetInputMode(CursorMode, CursorHidden)
-		} else {
-			w.view().SetInputMode(CursorMode, CursorNormal)
-			w.SetCursor(rawCursor)
+		if view := w.view(); view != nil { // not yet visible? linux weirdness
+			if rawCursor == nil {
+				view.SetInputMode(CursorMode, CursorHidden)
+			} else {
+				view.SetInputMode(CursorMode, CursorNormal)
+				w.SetCursor(rawCursor)
+			}
 		}
 		w.setCustomCursor(rawCursor, isCustomCursor)
 	}

